guides/20240823_how_to_setup_postgresql_playground_in_daytona.md

```diff
@@ -0,0 +1,372 @@
+  
+- ### Create a Table
+  **Example**: The SQL code below creates a table named `users`.
+
+  ```sql
+  CREATE TABLE users (
+      id SERIAL PRIMARY KEY,
+      name VARCHAR(100) NOT NULL,
+      email VARCHAR(255) UNIQUE NOT NULL,
+      created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+  );
+  ```
+
+  Your output should display similar one as mine
+
+  ![screenshoot of creating a database](assets/20240823_how_to_setup_postgresql_playground_in_daytona_img_4.jpg)
+
+
+- ### Insert a Record
+  **Example**: The SQL codes insert different records into the table `users` that was created by us above
+
+  ```sql
+  INSERT INTO users (name, email) VALUES ('Mark Zuckerberg', 'zuck@fb.com');
+  INSERT INTO users (name, email) VALUES ('Elon Musk', 'info@telsa.com');
+  INSERT INTO users (name, email) VALUES ('Bill Gates', 'bill.gates@gatesfoundation.org');
+  INSERT INTO users (name, email) VALUES ('Jack Dorsey', 'jack.dorsey@gmail.com');
+  ```
+
+  You should have similar screen like this below
+
+  ![screenshot of inserting records to table](assets/20240823_how_to_setup_postgresql_playground_in_daytona_img_5.jpg)
+
+- ### Read Data from Table
+  **Example**: This SQL query selects all records in the table `users` and return them.
+
+  ```sql
+  SELECT * FROM users WHERE name = 'Mark Zuckerberg';
+  ```
+
+  Your output should display something similar
+
+  ![screenshot of selecting a record from the table](assets/20240823_how_to_setup_postgresql_playground_in_daytona_img_6.jpg)
+
+- ### Update a Table Record
+  **Example**: This SQL query updates the table `users` record where `id` is 1.
+
+  ```sql
+  UPDATE users SET email = 'johndoe@email.com' WHERE id = 2;
+  ```
+
+  You output should be similar
+
+  ![screenshot of updating a record in the table](assets/20240823_how_to_setup_postgresql_playground_in_daytona_img_7.jpg)
+
+- ### Delete a Table Record
+  **Example**: This SQL code deletes the record where `id` is 2.
+
+  ```sql
+  DELETE FROM users WHERE id = 2;
+  ```
+
+  You should have similar output
+
+  ![screenshot of deleting a record from the table](assets/20240823_how_to_setup_postgresql_playground_in_daytona_img_8.jpg)
+
+  Run the SQL code below to check the table
+
+  ```sql
+  SELECT * FROM users;
+  ```
+
+  You should now see the deleted record is gone
+
+  ![screenshot of querying the table for all records](assets/20240823_how_to_setup_postgresql_playground_in_daytona_img_9.jpg)
+
+## Conclusion
+
+By following the steps above you should have learned how to set up a working PostgreSQL playground running on Daytona for you to start building or practice with PostgreSQL. From here, you could continue to explore the opportunities of using Daytona as your dev environment that suits your needs.
+
+## References
+
+*[Daytona](https://daytona.io)*
+
+*[PostgreSQL](https://postgresq.ort)*
+
+*[DevContainer](https://containers.dev)*
+
+<!-- Note on Definitions -->
```

Comment:
please remove this and implement as instructed in the note

Reply:
for example postgresql

Reply:
I resolved it